configs/segmenter/README.md

```diff
@@ -0,0 +1,54 @@
+# Segmenter: Transformer for Semantic Segmentation
+
```

Comment:
README files of model need to add abstract and image in paper which illustrates the algorithm.
Here is an example: 
https://github.com/open-mmlab/mmsegmentation/blob/master/configs/segformer/README.md